Libraries/Interaction/InteractionManager.js

```diff
@@ -73,13 +73,14 @@ var InteractionManager = {
   /**
    * Schedule a function to run after all interactions have completed.
    */
-  runAfterInteractions(callback: Function) {
-    invariant(
-      typeof callback === 'function',
-      'Must specify a function to schedule.'
-    );
-    scheduleUpdate();
-    _queue.push(callback);
+  runAfterInteractions(callback: ?Function): Promise {
+    return new Promise((resolve, reject) => {
+      scheduleUpdate();
+      if (callback) {
+        _queue.push(callback);
+      }
+      _queue.push(resolve);
+    });
```

Comment:
Yeah what @ide says. I didn't change the semantics around how `callback` gets executed. If you read the red portions of the diff, you'll see that it currently gets pushed onto the queue just like it does after my change.